test/bot-events.test.js

```diff
@@ -141,4 +141,47 @@ describe('Bot Events', function() {
     secondCall.args[0].should.equal('Channel not found in config, not joining:');
     secondCall.args[1].should.equal(channel);
   });
+
+  it('should send joins to slack', function() {
+    var a = this.bot.nickname;
```

Comment:
The tests would be a lot more readable if you use more descriptive variable names instead of `a`, `b` and `c`.